consensus/byzantine_test.go

```diff
@@ -420,7 +420,7 @@ func TestByzantineConflictingProposalsWithPartition(t *testing.T) {
 	// wait for someone in the big partition (B) to make a block
 	<-blocksSubs[ind2].Out()
 
-	t.Log("A block has been committed. Healing partition")
+	t.Logf("A block has been committed. Healing partition")
```

Comment:
```suggestion
	t.Log("A block has been committed. Healing partition")
```
Not sure why `Logf` is preferred here. We don't use a format verb?